### PR TITLE
Document how to use angle brackets

### PR DESCRIPTION
**Pull request by himynameisjonas**
Document what we decided on our [last tech/design vision meeting](https://paper.dropbox.com/doc/TechDesign-Vision-254-2019-uU8J1n9VUzlklsBcAzdd9) (back in April!!)

[RENDERED](https://deploy-preview-26--tt-code-style.netlify.com/ember/ember-angle-brackets)

### CODE REVIEW

File: content/ember/angle-brackets.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: Use angle bracket component syntax for all new components
+language: ember
+---
+
+But don't use the new syntax for components that already exists with the old syntax, i.e one syntax per component
+
+```handlebars
+  {{! Good }}
+  <EditPage @page={{page}} class="u-display--flex" />
```
Comment (emilkarl) [inline content/ember/angle-brackets.md]: should it be `class` or `@class`?

Comment (himynameisjonas) [reply content/ember/angle-brackets.md]: regular html attributes is defined without `@`

Comment (bzf) [reply content/ember/angle-brackets.md]: `class` since it's an HTML attribute and not a property passed to the component

Comment (himynameisjonas) [reply content/ember/angle-brackets.md]: That is one of the benefits with angle brackets

Comment (emilkarl) [reply content/ember/angle-brackets.md]: Nice!

**Pull request merged**